includes/Core/Modules/Modules.php

```diff
@@ -218,8 +218,6 @@ function( Module $module ) {
 			}
 		);
 
-		$this->sharing_settings->register();
```

Comment:
@kuasha420 this is the correct place where the instance of the module sharing settings class should be registered. Please, revert this change and remove the registration from the Plugins class.